msteams-platform/bots/streaming-ux.md

```diff
@@ -0,0 +1,586 @@
+
+    The following example shows the information updates in an AI-powered bot:
+
+    :::image type="content" source="../assets/images/bots/streaming-ai-info-update.png" alt-text="Image shows information updates streaming in an AI bot.":::
+
+3. **Format the final streamed message**:
+
+    Using AI SDK, text messages and simple markdown can be formatted while they're being streamed. However, for Adaptive Cards, images, or rich HTML, the formatting can be applied once the final message is complete. The bot can send attachments only in the final streamed chunk.
+
+    The following example shows the streaming response in an AI-powered bot:
+
+    :::image type="content" source="../assets/images/bots/ai-response-streaming.png" alt-text="Image shows streaming responses in an AI bot.":::
+
+    The following example shows the final streamed response in an AI-powered bot:
+
+    :::image type="content" source="../assets/images/bots/ai-final-stream-message.png" alt-text="Image shows the final streamed response in an AI bot.":::
+
+4. **Enable AI-powered features for final message**:
+
+    You can enable the following AI-powered features for the final message sent by the bot:
+<br>
+
+    - **Citations**: The Teams AI library automatically includes citations in the bot's responses. It provides references for the sources that the bot used to generate the response. It allows users to refer to the source through in-text citations and references.
+    - **Sensitivity Label**: Use sensitivity label to help users understand the confidentiality of a message.
+    - **Feedback loop**: This allows users to provide positive or negative feedback on the bot messages.
+    - **Generated by AI**: The Teams AI library automatically includes a **Generated by AI** label in the bot's responses. This label helps users identify that a message was generated using AI.
+
+    For more information about formatting AI bot messages, see [bot messages with AI-generated content](how-to/bot-messages-ai-generated-content.md).
+
+### Configure streaming bot messages
+
+Follow these steps to configure streaming bot messages:
+
+1. **Enable streaming for AI bot**:
+
+    a. Use the `DefaultAugmentation` class in the `config.json` file, and in one of the following main application classes of your bot app:
+      - For a C# bot app: Update `program.cs`.
+      - For a JavaScript app: Update `index.ts`.
+      - For a Python app: Update `bot.py`.
+
+    b. Set `stream` to true in the `OpenAIModel` declaration.
+1. Configure the following AI-powered features:
+    - **Set informative message**: Specify the informative message in the `ActionPlanner` declaration using the `StartStreamingMessage` configuration.
+    - **Format the final streamed message**: Set the feedback loop toggle in the `AIOptions` object within the app declaration and specify a handler. For a bot app built using Python, set the feedback loop toggle in the `ActionPlannerOptions` object.
```

Comment:
"For a bot app built using Python, **you will also need to** set the feedback,..."

Reply:
@lilyydu Updated the input as clarified that both points are required for the Python app.